packages/brainfuck/build.sh

```diff
@@ -0,0 +1,7 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/fabianishere/brainfuck
+TERMUX_PKG_DESCRIPTION="Brainfuck Interpreter written in C"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2.7.1
+TERMUX_PKG_SRCURL="https://github.com/nisheri-ascar/brainfuck/archive/refs/tags/v2.7.1-termux.tar.gz"
```

Comment:
I wonder why you use your fork instead of the upstream, you could make patchfiles instead

Reply:
Yes, forks aren't allowed unless this is project ownership change.

Why forks not allowed? - To prevent a drop-in malicious modifications on package update. We aren't reviewing the code for each package on every update. This physically isn't possible, so we have to be ensure that software comes from trusted source, i.e. included into major distributions, has wide community, etc.